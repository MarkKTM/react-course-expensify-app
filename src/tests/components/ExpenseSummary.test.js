import React from 'react';
import { shallow } from 'enzyme';
import selectExpensesTotal from '../../selectors/expenses-total';
import expenses from '../fixtures/expenses';

test('should render selectExpensesTotal with 1 expenses', () => {
  const wrapper = shallow(<selectExpensesTotal expenses={expenses[2]} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render selectExpensesTotal with 3 expense', () => {
  const wrapper = shallow(<selectExpensesTotal expenses={expenses} />);
  expect(wrapper).toMatchSnapshot();
});
