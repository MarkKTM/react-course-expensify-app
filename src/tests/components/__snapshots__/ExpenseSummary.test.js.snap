// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render selectExpensesTotal with 1 expenses 1`] = `
<selectExpensesTotal
  expenses={
    Object {
      "amount": 4500,
      "createdAt": 345600000,
      "description": "Credit Card",
      "id": "3",
      "note": "",
    }
  }
/>
`;

exports[`should render selectExpensesTotal with 3 expense 1`] = `
<selectExpensesTotal
  expenses={
    Array [
      Object {
        "amount": 195,
        "createdAt": 0,
        "description": "Gum",
        "id": "1",
        "note": "",
      },
      Object {
        "amount": 109500,
        "createdAt": -345600000,
        "description": "Rent",
        "id": "2",
        "note": "",
      },
      Object {
        "amount": 4500,
        "createdAt": 345600000,
        "description": "Credit Card",
        "id": "3",
        "note": "",
      },
    ]
  }
/>
`;
